packages/relay/tests/lib/eth/eth_sendRawTransaction.spec.ts

```diff
@@ -127,6 +128,13 @@ describe('@ethSendRawTransaction eth_sendRawTransaction spec', async function ()
         balance: Hbar.from(100_000_000_000, HbarUnit.Hbar).to(HbarUnit.Tinybar),
       },
     };
+    const RECEIVER_ACCOUNT_RES = {
+      account: ACCOUNT_ADDRESS_1,
+      balance: {
+        balance: Hbar.from(100_000_000_000, HbarUnit.Hbar).to(HbarUnit.Tinybar),
```

Comment:
Just to be on the same page. The mirror node response for balance is denominated in tinybars (not weibars). 

Here we mock `100_000_000_000` hbars which is `100_000_000_000 * 100_000_000 * 10_000_000_000` weibars in the ethers world.

E.g. for that test I would mock 1 hbar
```typescript
balance: Hbar.fromTinybars(100_000_000).to(HbarUnit.Tinybar)
// or
balance: Hbar.from(1, HbarUnit.Hbar).to(HbarUnit.Tinybar)
```